tests/test_models.py

```diff
@@ -1,15 +1,6 @@
 import pytest
 
-from transmogrifier.models import (
-    AlternateTitle,
-    Contributor,
-    Date,
-    DateRange,
-    Identifier,
-    Link,
-    Note,
-    Subject,
-)
```

Comment:
Allows us to easily use `timdex.<field>` without having to explicitly import each field class, similar to what we do for other test modules.